app/services/state_file/state_information_service.rb

```diff
@@ -76,8 +76,8 @@ def state_intake_class_names
         submission_builder_class: SubmissionBuilder::Ty2022::States::Az::AzReturnXml,
         survey_link: "https://codeforamerica.co1.qualtrics.com/jfe/form/SV_0v0BnYNRoLIqzhY",
         submission_type: "Form140",
+        tax_payment_info_text: "https://azdor.gov/make-payment-online",
         tax_payment_info_url: "https://azdor.gov/making-payments-late-payments-and-filing-extensions",
```

Comment:
based on the `app/views/state_file/questions/taxes_owed/edit.html.erb` implementation, looks like `tax_payment_url` is the link text and `tax_payment_info_url` is the actual link we want to navigate to. This was not being used in the return_status, so making that consistent 

[Q here](https://codeforamerica.atlassian.net/browse/FYST-1442?focusedCommentId=14353)